exist-core/src/test/xquery/xquery3/flwor.xql

```diff
@@ -156,3 +156,17 @@ function flwor:allowing-empty($n as xs:integer) {
     return concat("[", $x, "]")
 };
 
+declare
```

Comment:
Do we have a test that shows the correct behaviour if `allowing empty` is not set?

Reply:
There isn't any right now
Should I add them?